comms/DiskRepository.cpp

```diff
@@ -716,10 +717,13 @@ class CDiskRepository : public CRepositoryBase
                 }
                 CComPtr<IAttribute> attribute = CreateDiskAttribute(this, toModule, itr->get()->GetLabel(), itr->get()->GetType()->GetRepositoryCode(), toPath, itr->get()->GetText(), false);
                 attribute->Refresh(false, attribute);
-                return true;
+                status = true;
+            }
+            else {
+                status = false;
             }
         }
-        return false;
+        return status;
```

Comment:
Might be better to return an array of failures, then display a message?

Reply:
@GordonSmith did think of that as well. Can do...